daemon/logs.go

```diff
@@ -21,13 +21,16 @@ import (
 // ContainerLogs hooks up a container's stdout and stderr streams
 // configured with the given struct.
 func (daemon *Daemon) ContainerLogs(ctx context.Context, containerName string, config *backend.ContainerLogsConfig, started chan struct{}) error {
+	if !(config.ShowStdout || config.ShowStderr) {
+		return fmt.Errorf("You must choose at least one stream")
+	}
```

Comment:
Minor optimization; I thought it was better to error out early, and move this before `daemon.GetContainer(containerName)`, which is a more heavyweight action.